resources/assets/js/markdown-editor/markdown-editor.js

```diff
@@ -0,0 +1,293 @@
+import "./css/style.css"; // Editor's Style
+
+import Editor from "@toast-ui/editor";
+
+import {
+    simplecastPlugin,
+    youtubePlugin,
+    twitterPlugin,
+    undoPlugin,
+    redoPlugin,
+    underlinePlugin,
+    headingPlugin,
+    previewPlugin,
+    referencePlugin,
+    alertPlugin,
+    linkCollectionPlugin,
+    embedLinkPlugin,
+} from "./plugins/index.js";
+
+import { extractTextFromHtml } from "./utils/utils.js";
+
+const AVERAGE_WORDS_READ_PER_MINUTE = 200;
+
+const MarkdownEditor = (height = null, toolbar = "basic", extraData = {}) => ({
+    editor: null,
+    toolbar: null,
+    toolbarItems: null,
+    showOverlay: false,
+    charsCount: 0,
+    wordsCount: 0,
+    readMinutes: 0,
+    height: height !== null ? height : "600px",
```

Comment:
should be able to use `height || '600px'` here so it defaults to 600px if height is falsy. Ignore this if you want `height=0` to be a valid use case (since 0 is falsy too)